docker/test/docker-compose.yml

```diff
@@ -17,8 +17,9 @@ services:
       - APP_SETTINGS=Testing
       - FLASK_APP=manage.py
       - COVERALLS_REPO_TOKEN=${COVERALLS_REPO_TOKEN}
+      - SLACK_API_TOKEN=${SLACK_API_TOKEN}
     # env_file:
```

Comment:
Could this be deleted if it's not being used?

Reply:
This is being used. If we delete it tests will fail

Reply:
Is there any reason why you've commented it out?

Reply:
The `env_file` is used locally to load the local environment. It is commented out to ease the task of anyone using the local env. `just uncommenting and moving on` instead of configuring the file again.